pyrefly/lib/test/lsp/code_actions.rs

```diff
@@ -236,3 +236,66 @@ my_export
         report.trim()
     );
 }
+
+#[test]
+fn test_import_from_stdlib() {
```

Comment:
this test is great, i'd like to keep it 

the one concern would be that different python / typeshed versions change where `TypeVar` comes from, but since we bundled typeshed it should match on all systems. an integration test of this behavior (where we actually use system python) would be less ideal than what you did.